ix-dev/community/codegate/templates/docker-compose.yaml

```diff
@@ -0,0 +1,31 @@
+{% set tpl = ix_lib.base.render.Render(values) %}
+
+{% set c1 = tpl.add_container(values.consts.codegate_container_name, "image") %}
+{% set perm_container = tpl.deps.perms(values.consts.perms_container_name) %}
+{% set perm_config = {"uid": values.run_as.user, "gid": values.run_as.group, "mode": "check"} %}
+
+{% do c1.set_user(values.run_as.user, values.run_as.group) %}
+{% do c1.healthcheck.set_test("wget", {"port": values.network.api_port.port_number, "path": "/health"}) %}
+
+{% do c1.add_port(values.network.web_port) %}
+{% do c1.add_port(values.network.api_port) %}
+{% do c1.add_port(values.network.proxy_port) %}
```

Comment:
Are the internal ports changeable? 
eg via env variable? If not, you need set the `, {"container_port": ...}` on each one (Also on healthcheck port, you need to pass the internal one)

If they are changeable, I'd prefer do that instead.